app/Email.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace emailApp;
+
+class Email {
+	function validateEmail($email) {
```

Comment:
There should be comment explaining purpose of function is phpdocs format. http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_phpDocumentor.howto.pkg.html